Makefile

```diff
@@ -7,10 +7,8 @@ release: generate
 	cd .. && tar -zcvf dokku-$(shell cat meta/main.yml | grep version | head -n 1 | cut -d':' -f2 | xargs).tar.gz dokku
 
 .PHONY: generate
-generate: clean README.md defaults/main.yml
+generate: clean README.md
 
+.PHONY: README.md
 README.md:
 	bin/generate
-
-defaults/main.yml:
```

Comment:
I think we still want this, and it should mirror the README.md command.

Reply:
makes sense  - I overlooked that it also touches the yml file